src/tests/Interop/PInvoke/Generics/GenericsNative.Vector256B.cpp

```diff
@@ -8,9 +8,14 @@
 
 #if defined(TARGET_XARCH)
     #include <immintrin.h>
+#elif defined(TARGET_ARMARCH)
```

Comment:
My only feedback would be to add a comment in all files in the else branch for `TARGET_ARMARCH`. The comment should state `// Intentionally empty`